db/base.py

```diff
@@ -7,3 +7,11 @@
 
 def uuid_str():
     return str(uuid4())
+
+
+def total_size():
+    return 200
+
+
+def sector_size():
+    return 10
```

Comment:
Might be better as a Global variable.

Not for this PR, but in the future, we need to make sure that the entire DB is deleted or migrated to use new sector size if the sector size is changed.